.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "AmpliTF"]
+	path = AmpliTF
+	url = https://github.com/apoluekt/AmpliTF.git
```

Comment:
A Git submodule is not a good solution in Python development, so this should be removed in the future once/if `amplitf` becomes a package.

Reply:
> Note that the submodule currently points to a a fork, because some changes have not been merged into the master. This should be relinked later on.

Is this remark outdated? apoluekt is the upstream, right?

Reply:
Yes

Reply:
Actually I'm not sure how `python setup.py install` handles this (non dev mode). Because this should be installed into a separate dir of the venv, in order to be found via `import amplitf`

Reply:
Yeah should be solved soon... better to create an issue for this after this PR is merged (which is still necessary to continue development)
https://github.com/ComPWA/tensorwaves/pull/35#discussion_r414524210